eng/scripts/Create-Apiview-Token-Files.ps1

```diff
@@ -0,0 +1,39 @@
+[CmdletBinding()]
+param (
+  [Parameter(Mandatory = $true)]
+  [string]$ReviewDetailsJson,
+  [Parameter(Mandatory = $true)]
+  [string]$StagingPath,
+  [Parameter(Mandatory = $true)]
+  [string]$WorkingDir,
+  [Parameter(Mandatory = $true)]
+  [string]$StorageBaseUrl,
+  [Parameter(Mandatory = $true)]
+  [string]$ContainerSas,
+  [Parameter(Mandatory = $true)]
+  [string]$Language
+)
+
+
+Write-Host "Review Details Json: $($ReviewDetailsJson)"
+$reviews = ConvertFrom-Json $ReviewDetailsJson
+if ($reviews -ne $null)
+{
+    foreach($r in $reviews)
+    {
+        $codeDir = New-Item -Path $WorkingDir/$($r.ReviewID)/$($r.RevisionID) -ItemType Directory
+        $sourcePath = $StorageBaseUrl + "/" + $r.FileID + $ContainerSas
+        Write-Host "Copying $($sourcePath)"
+        azcopy cp "$sourcePath" $codeDir/$($r.FileName) --recursive=true
+
+        #Create staging path for review and revision ID
+        New-Item -Path $StagingPath/$($r.ReviewID)/$($r.RevisionID) -ItemType Directory
```

Comment:
Will this never happen already exist? If so this new-item will fail.

Reply:
Added test-path to make sure before I create it. It's unlikely to have this directory exists though because we create per each revision ID under each review and these are GUIDs.